packages/hebao_v2/contracts/base/SmartWallet.sol

```diff
@@ -142,6 +146,24 @@ contract SmartWallet is ERC1271
     {
     }
 
+    function getOwner()
+        public
+        view
+        override
+        returns (address)
+    {
+        return wallet.owner;
+    }
+
+    function getCreationTimestamp()
```

Comment:
how about `owner()` and `creationTimestamp()`?

Reply:
If I do this I get these shadows warning:
```
contracts/iface/ILoopringWalletV2.sol:19:9: Warning: This declaration shadows an existing declaration.
        address             owner,
        ^-----------------------^
contracts/iface/ILoopringWalletV2.sol:40:5: The shadowed declaration is here:
    function owner()
    ^ (Relevant source part starts here and spans across multiple lines).

contracts/base/SmartWallet.sol:110:9: Warning: This declaration shadows an existing declaration.
        address             owner,
        ^-----------------------^
contracts/base/SmartWallet.sol:149:5: The shadowed declaration is here:
    function owner()
    ^ (Relevant source part starts here and spans across multiple lines).
```

Probably because these kinds of getters get automatically generated when there are state variable that are called like that, so even though there is no problem it seems like it's better not to do this to not run into these kinds of automatic checks.